src/platform/packages/shared/kbn-ebt-tools/src/performance_metrics/context/measure_interaction/index.ts

```diff
@@ -36,7 +36,7 @@ export function measureInteraction(pathname: string) {
       let performanceMeta: PerformanceMeta | undefined;
       performance.mark(perfomanceMarkers.endPageReady);
 
-      if (eventData?.meta) {
+      if (eventData?.meta?.rangeFrom && eventData?.meta?.rangeTo) {
```

Comment:
Maybe I'm missing something but here I think you won't be adding `description` when `rangeFrom` or `rangeTo` are undefined. I think we should handle description separately from those two values.

Reply:
true 🤦‍♂️ Thank you!